services/core-web/src/components/mine/Incidents/MineIncidentTable.tsx

```diff
@@ -84,6 +82,21 @@ const MineIncidentTable = (props) => {
     pageData,
   } = props;
 
+  const handleNavigateToIncident = async (record) => {
+    const editIncident = incidents.find((incident) => incident.mine_incident_guid === record.key);
+    await dispatch(storeMineIncident(editIncident));
+
+    browserHistory.replace(
+      router.VIEW_MINE_INCIDENT.dynamicRoute(
+        editIncident.mine_guid,
+        editIncident.mine_incident_guid
+      )
+    );
+
+    // This is a temp fix to force the page to reload and prevent a stale data issue when deleting incident documents
```

Comment:
This is the slightly janky fix for the actual issue.  It forces a reload before navigating so it ensures stale data is eliminated.

Booo.  Wanted a better solution, but can't spend more time on this right now.